earth_enterprise/src/common/timeutils.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 The Open GEE Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <time.h>
+#include <ostream>
+
+#include "khTypes.h"
+#include "timeutils.h"
+
+using namespace std;
+
+namespace getime {
+
+const u_int32_t SEC_TO_NSEC = 1e9;
```

Comment:
They're essentially equivalent, but isn't uint32_t generally considered "the standard way" to go about things?

Reply:
I was going based off of what is defined in khTypes.h, but I like uint32_t better. I'll switch to that.